### PR TITLE
Gantt: describe export fonts

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

**Comment by arminal** [comment]
#GrammarReview

Short document, no edits.

File: api-reference/50 Common/Object Structures/PdfExportGanttProps/dateRange.md [hunk 1]
```diff
@@ -8,3 +8,23 @@ type: String | Object
...
+        dateRange: 'visible'
+
+- A range of task indexes.
+
+        dateRange: {
+            startIndex: 0,
+            endIndex: 5
+        }
+
+- A range of [dates](https://www.w3schools.com/js/js_date_methods_set.asp).
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/PdfExportGanttProps/dateRange.md]: - External links should be opened in a new tab.
- A list item doesn't need a dot in the end if it isn't a sentence.
- Do you mean dates or values of Date type?

File: api-reference/50 Common/Object Structures/PdfExportGanttProps/format.md [hunk 2]
```diff
@@ -8,3 +8,20 @@ default: undefined
...
 
 ---
+
+The **format** property accepts the following formats: 
+
+- Standard document sizes from A0 to A10.
+
+        format: 'a4'
+
+- Custom paper width and height.
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/PdfExportGanttProps/format.md]: Paper?

File: api-reference/50 Common/utils/pdfExporter/exportGantt(options).md [hunk 3]
```diff
@@ -24,15 +24,14 @@ This method requires the <a href="https://github.com/MrRio/jsPDF" target="_blank
...
+- [format](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#format) - Specifies the document size.
+- [landscape](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#landscape) - Specifies whether to use horizontal orientation for the document.
+- [fileName](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#fileName) - Specifies the file name.
+- [exportMode](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#exportMode) - Specifies the part of the component to export (chart area, tree list area, or the entire component).
+- [font](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/font/) - Specifies the font.
+- [dateRange](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#dateRange) - Restricts data output against a specified date range.
+- [margins](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#margins) - Specifies the outer indents of the exported area.
```
Comment (RomanTsukanov) [inline api-reference/50 Common/utils/pdfExporter/exportGantt(options).md]: You can simply give a link to the PdfExportGanttProps docs. This way you wouldn't have to update this list if PdfExportGanttProps will be added another prop.

File: api-reference/_hidden/PdfExportGanttFont/PdfExportGanttFont.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+id: PdfExportGanttFont
+---
+---
+##### shortDescription
+Configures a custom font used for the Gantt data export.
+
+---
+
+The exporter supports standard PDF fonts. Specify the [font](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#font) property in the [exportGantt](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportGanttoptions) method to use a custom font in the exported Gantt data.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/PdfExportGanttFont/PdfExportGanttFont.md]: The paragraph starts with standard fonts, but then you talk about custom fonts.

File: api-reference/_hidden/PdfExportGanttFont/PdfExportGanttFont.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+    <!-- tab: JavaScript -->
+    function exportGantt() {
+        DevExpress.pdfExporter.exportGantt({
```
Comment (RomanTsukanov) [inline api-reference/_hidden/PdfExportGanttFont/PdfExportGanttFont.md]: This isn't framework-specific. It's better to give the example similarly to [this article](https://js.devexpress.com/Documentation/ApiReference/Common/Utils/utils/#getTimeZonesdate).

File: api-reference/_hidden/PdfExportGanttFont/PdfExportGanttFont.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+            // ...
+            font: {
+                fontObject: test_font,
```
Comment (RomanTsukanov) [inline api-reference/_hidden/PdfExportGanttFont/PdfExportGanttFont.md]: What's a test_font?

File: api-reference/_hidden/PdfExportGanttFont/fontObject.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+type: Object
+default: undefined
+---
+---
+##### shortDescription
+A custom font object.
+
+---
+
+Use the <a href="https://rawgit.com/MrRio/jsPDF/master/fontconverter/fontconverter.html" target="_blank">jsPDF font converter</a> to generate a custom font object. Specify font settings and load your font file (.ttf). The converter generates a .js file with the content of your font file.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/PdfExportGanttFont/fontObject.md]: What do I do with the generated js file?

File: api-reference/_hidden/PdfExportGanttFont/style.md [hunk 8]
```diff
@@ -5,7 +5,8 @@ default: undefined
...
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The font style.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Acceptable property values: *"normal"*, *"bold"*, and *"italic"*.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/PdfExportGanttFont/style.md:12]: This kind of information is usually specified via the doccomments.

File: api-reference/50 Common/Object Structures/PdfExportGanttProps/dateRange.md [hunk 9]
```diff
@@ -8,3 +8,23 @@ type: String | Object
...
+        dateRange: 'visible'
+
+- A range of task indexes
+
+        dateRange: {
+            startIndex: 0,
+            endIndex: 5
+        }
+
+- A range of <a href="https://www.w3schools.com/js/js_dates.asp" target="_blank">Date</a> values.
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/PdfExportGanttProps/dateRange.md]:
```suggestion
- A range of <a href="https://www.w3schools.com/js/js_dates.asp" target="_blank">Date</a> values
```

File: api-reference/50 Common/Object Structures/PdfExportGanttProps/format.md [hunk 10]
```diff
@@ -8,3 +8,20 @@ default: undefined
 Specifies the document size.
 
 ---
+
+The **format** property accepts the following formats: 
+
+- Standard document sizes from A0 to A10.
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/PdfExportGanttProps/format.md]:
```suggestion
- Standard document sizes from A0 to A10
```

File: api-reference/50 Common/Object Structures/PdfExportGanttProps/format.md [hunk 11]
```diff
@@ -8,3 +8,20 @@ default: undefined
...
+
+        format: 'a4'
+
+- Custom document width and height.
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/PdfExportGanttProps/format.md]:
```suggestion
- Custom document width and height
```

File: api-reference/50 Common/Object Structures/PdfExportGanttProps/format.md [hunk 12]
```diff
@@ -8,3 +8,20 @@ default: undefined
...
+
+        format: {
+            width: 210,
+            height: 300
+        }
+
+- Auto. Gantt adjusts document size to the exported data.
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/PdfExportGanttProps/format.md]:
```suggestion
- Auto (Gantt adjusts document size to the exported data) 
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**